scripts/plugins/install_plugins.py

```diff
@@ -32,7 +32,7 @@ def install_plugins():  # pylint: disable=too-many-locals,too-many-statements
         bitops_build_configuration.bitops.plugins, None
     )
 
-    plugin_dir = BITOPS_plugin_dir
+    installed_plugins_dir = BITOPS_installed_plugins_dir
```

Comment:
Same in this case.
```suggestion
```
We can just use the original `BITOPS_installed_plugins_dir` where needed instead of creating a dedicated new variable.